docs/developer/taxes.mdx

```diff
@@ -394,3 +393,38 @@ mutation {
 ```
 
 For orders, pass the order ID as an argument and adjust the query to fetch the order data in the `taxableObject` field. To re-enable tax charging, pass `taxExemption: false`.
+
+## Address for tax calculation
+
+The process of determining the country code follows this sequence:
+
+1. Shipping address
+2. Billing address
+3. The channel's default country
+
+|             | Store Address | Shipping Address | Billing Address | Warehouse Address |
```

Comment:
I understand the sequence above, but I don't know what the table represents.  Does it represent that each address can have different country? Maybe worth adding some description or something?

Reply:
Ohh, it is clear after checking the examples. Maybe worth adding a note to the table, that this is a helper table for examples?

Reply:
note added 625f505696e2c6ae47d38f3d0b496a32ad451d2f